doc/RecordingControl.xml

```diff
@@ -2301,12 +2301,61 @@ secfrac = "." 1*6DIGIT</programlisting>
+            <row>
+              <entry>
+                <para>
+                  <literal>Timescale</literal>
+                </para>
+              </entry>
+              <entry>
+                <para>The <literal>timescale</literal> (or clock) from the segment in hertz</para>
+              </entry>
+              <entry>
+                <para>Integer (See <xref linkend="_refTimescaleAttribute"/>)</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para>
+                  <literal>SeekTable</literal>
+                </para>
+              </entry>
+              <entry>
+                <para>Seek table to efficiently locate file fragments and seek in the file</para>
+              </entry>
+              <entry>
+                <para>String (See <xref linkend="_refSeekTableAttribute"/>)</para>
+              </entry>
+            </row>
           </tbody>
         </tgroup>
       </table>
       <para>For storage target systems that do not allow attribute changes during write the
         attribute Duration should be omitted when the total duration of the file may change while it
         is written e.g. due to an override segment duration request.</para>
+      <section xml:id="_refTimescaleAttribute">
+        <title>Timescale Attribute</title>
+        <para>
+          The value of the <literal>timescale</literal> property of the <literal>MediaHeaderBox</literal> in the segment as defined in ISO/IEC 14496-12:2012 section 8.4.2. If there are multiple tracks in the segment each of these boxes shall all have the same <literal>timescale</literal> value. The <literal>MovieHeaderBox</literal> shall also have the same <literal>timescale</literal> value.
+        </para>
+      </section>
+      <section xml:id="_refSeekTableAttribute">
+        <title>SeekTable Attribute</title>
+        <para>
+          This seek table attribute is a string that contains the 'mfra' (Movie Fragment Random Access) box corresponding to the associated ISO file. It is encoded in base64 with padding according to RFC 4658. The binary format of the box is defined in ISO/IEC 14496-12:2012 section 8.8.9. This optional box has to be the last one at the end of the file. The optionally condensed copy allows the receiver to efficiently locate file fragments and seek in the file without the need to access the file itself.
+        </para>
+        <para>
+          The size of the SeekTable should be limited to a reasonable access precision in the order of single digit seconds. Additionally the SeekTable should contain more than twenty entries. When the SeekTable attribute is present it shall contain at least one entry for the very first sample of each track. The mfro box may be omitted in the SeekTable as it doesn't add any value, but the mfra length shall be updated according. The Timescale attribute shall be provided if this attribute is present, as it is necessary the interpret the SeekTable.
```

Comment:
>The mfro box may be omitted in the SeekTable as it doesn't add any value, but the mfra length shall be updated according

The mfro box may be omitted in the SeekTable as it doesn't add any value, but the mfra length shall be updated **accordingly**